### PR TITLE
Branch1: 

### PR DESCRIPTION
**Pull request by patrickjcolgan**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -182,12 +191,15 @@ var NoteTracker = {
...
+    // $('#noteList').append('');
   },
   clearForm: function () {
     document.getElementById('displayWindow').innerHTML = '';
+    // $('#displayWindow').append('');
   },
   clearNoteWrapper: function (){
     document.getElementById('noteWrapper').innerHTML = '';
+    // $('#noteWrapper').append('');
   },
```
Comment (meganwalter) [inline app.js:203]: The Jquery command here would be `.html('')`

File: app.js [hunk 2]
```diff
@@ -24,24 +24,27 @@ function User (username, password, library, tagLibrary) {
...
+  // document.getElementById('loginForm').innerHTML = '';
+  $('#loginForm').append('<form name="loginform" class="whiteText" id="returnUser"><fieldset><legend>Returning User</legend><label>Username</label><input class="labelColor" type="text" name="usr" placeholder="username" required="required"><label>Password</label><input class="labelColor" type="password" name="pword" placeholder="password" required="required"><p id="msg"></p><input class="button-primary" type="submit" value="Login"></fieldset></form><input class="button-primary" type="submit" value="Create New User" id="newButton">');
   var returnUserEl = document.getElementById('returnUser');
```
Comment (meganwalter) [inline app.js:48]: What if we re-factored this blob of text to look nicer by entering new elements onto new lines?

File: app.js [hunk 3]
```diff
@@ -61,6 +64,12 @@ function newUser(event) {
       userExists = true;
```
Comment (meganwalter) [inline app.js:64]: how is this for loop better than the each loop they had? Wouldn't you want to use `forEach`? #functionalprogramming #ftw

File: app.js [hunk 4]
```diff
@@ -61,6 +64,12 @@ function newUser(event) {
...
     }
   }
+  // userLibrary.each(function() {
+  //   if (userLibrary[i].username === username) {
+  //     msg.textContent = "Username taken";
+  //     userExists = true;
+  //   }
+  // })
```
Comment (patrickjcolgan) [inline app.js:72]:
```
.forEach(function(ele) {
//some code using ele instead of userLibrary[i]
});
```

**Pull request merged**